### ISSUE TITLE
Weird Icons

### ISSUE THREAD

**Issue by wowmodder** [renzhn/ColdLauncher#6]
Hey, I'll start of by saying thanks for the amazing launcher you made, it's the best one on the playstore!
So I bought the full version so I can apply icon packs, but they always get messed up (i.e. the phone icon looks like the contacts icon or some of them don't even get modified).
I think what would fix this is if you could give an option to edit apps in the launcher, so we can modify the name and icon for each app, this way we can easily fix anything it gets wrong. Would that be possible?
Thanks for all the great work!

**Comment by renzhn** [comment]
Thanks for your feedback! The "some of them don't even get modified" issue is because it's not covered in the icon pack. And yes, I'm considering adding edit apps feature in the pop up menu.
Would you please post an image of the "messed up" issue if you have chance?

**Comment by wowmodder** [comment]
Yeah, but I can't find an icon pack that works for all my apps, that's why it would be great to be able to edit them manually:)
I'm not sure how to post an image in here so I hosted it, this link should work:

https://s24.postimg.org/jop8szfmt/Screenshot_2016_12_18_20_59_29_net_ohrz_coldlaun.png

This is just an example with one icon pack but it happens often with a couple apps. As you can see the phone and contacts look exactly the same. Even this is just a simple issue that could be fixed with manual editing of icons.

Anyway, thanks again for all the great work!
(P.S. would it be possible to be able to choose the folder type on launcher? Because they're different on the new update, they look great, but the other ones were cool too. Anyway, unless it's complicated to do, choosing folder type would be a pretty cool feature that I haven't seen on many launchers)

**Comment by renzhn** [comment]
The phone and contacts have the same icon is probably because actually they are in one package(apk), using a different icon pack might help.
I want to add editting app feature too, however I develop this app in my spare time and I'm still figuring it out.
